rasa/shared/nlu/training_data/message.py

```diff
@@ -105,7 +110,7 @@ def build(
         Returns:
             Message
         """
-        data = {TEXT: text}
+        data: Dict[Text, Any] = {TEXT: text}
```

Comment:
We don't usually define the types for local variables do we?

Reply:
it's needed because the typechecker cannot infer the type correctly: this line creates a dict of type `Dict[Text, Text]` but we are adding other values that have types other than `Text`